analytics-sidecar-vertx-client-shaded/build.gradle

```
@@ -75,6 +75,7 @@ shadowJar {
     relocate 'com.fasterxml.jackson', 'o.a.c.sidecar.client.shaded.com.fasterxml.jackson'
     relocate 'io.netty', 'o.a.c.sidecar.client.shaded.io.netty'
     relocate 'io.vertx', 'o.a.c.sidecar.client.shaded.io.vertx'
+    relocate 'org.apache.cassandra.sidecar.client', 'org.apache.cassandra.sidecar.analyticsclient'
```

for consistency we should use instead
```suggestion
relocate 'org.apache.cassandra.sidecar.client', 'o.a.c.sidecar.client.shaded.client'
```

---

Done